lib/ae_mdw_web/router.ex

```diff
@@ -57,6 +57,8 @@ defmodule AeMdwWeb.Router do
       get "/micro-blocks/:hash/transactions", TxController, :micro_block_txs
 
       get "/transactions", TxController, :txs
+      get "/transactions/count", TxController, :count
+      get "/transactions/count/:id", TxController, :count_id
```

Comment:
This didn't change, we should leave it unchanged

Reply:
I moved them because `/transactions/:hash` was above them and when i tried calling this it was trying to find a transaction with hash `count`

Reply:
This is already fixed here https://github.com/aeternity/ae_mdw/pull/1712, not yet merged, though